CI_YAML.md

```diff
@@ -124,33 +125,155 @@ following are a list of keys that are reserved for special use.
 ``` yaml
-$flutter/osx_sdk : >-
-  {
-    "sdk_version": "14e222b",
-    "runtime_versions":
-      [
-        "ios-16-4_14e222b",
-        "ios-16-2_14c18"
-      ]
-  }
+add_recipes_cq: "true"
 ```
+</td>
+  </tr>
+  <tr>
+    <td>cache_name</td>
+    <td>The name identifier of the second layer Engine source cache.
+    </td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
 
-**add_recipes_cq**: String boolean whether to add this target to flutter/recipes CQ. This ensures
-changes to flutter/recipes pass on this target before landing.
+``` yaml
+cache_name: "builder"
+```
+</td>
+  </tr>
+  <tr>
+    <td>cache_path</td>
+    <td>The paths of Engine checkout source that will be auto saved to CAS for boosting source checkout when
+caches no longer exist from the bots.
+    </td>
+    <td>N/A</td>
+    <td>list</td>
+    <td>
+
+``` yaml
+cache_paths: >-
+  [
+    "builder",
+    "git"
+  ]
+```
+</td>
+  </tr>
+  <tr>
+    <td>cache_root</td>
+    <td>The root of cache directory used in recipes: `api.path[cache_root]`.
+    </td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
 
-**dependencies**: JSON list of objects with "dependency" and optionally "version".
-The list of supported deps is in [flutter_deps recipe_module](https://cs.opensource.google/flutter/recipes/+/master:recipe_modules/flutter_deps/api.py).
+``` yaml
+cache_name: "cache"
+```
+</td>
+  </tr>
+  <tr>
+    <td>clobber</td>
+    <td>Whether to clean the Engine source code cache.
+    </td>
+    <td>"false"</td>
+    <td>string bool</td>
+    <td>
+
+``` yaml
+clobber: "true"
+```
+</td>
+  </tr>
+  <tr>
+    <td>config_name</td>
+    <td>The config name of the targets. It is used for <a href="https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2">`Engine V2 recipes`</a>,
+and is a one-on-one map to the config files located under <a href="https://github.com/flutter/engine/tree/main/ci/builders">`ci/builders`</a>. This is
+not needed for targets using none `Engine V2 recipes`.
+    </td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
+
+``` yaml
+config_name: linux_benchmarks
+```
+</td>
+  </tr>
+  <tr>
+  <td>contexts</td>
+  <td>The list of contexts that will guide <a href="https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/flutter_deps/api.py#665">recipes</a> to add to <a href="https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack">ExitStack</a>. This will initialize and prepare the virtual device used for tests. Other supported contexts include: `osx_sdk`, `depot_tools_on_path`, etc.
+  </td>
+  <td>N/A</td>
+  <td>list</td>
+  <td>
+  
+```yaml
+contexts: >-
+        [
+          "android_virtual_device"
+        ]
+```
+</td>
+  </tr>
+  <tr>
+    <td>cores</td>
+    <td>The machine cores a target will be running against. A higher number of cores may be needed for extensive targets.
+    <br>
+    Note: This property will be auto populated to CI builder dimensions, based on which CI determines which
```

Comment:
Consider rewording this to:

"based on which testbed the CI determines will run the target."

instead of:

based on which CI determines which testbed to run this target.

Reply:
Huh, I want to emphasize that the CI relies on the dimensions to determine the testbed. How about:
```
This property will be auto populated to CI builder dimensions, which CI uses to determine the testbed to run this target.
```